arkouda/pdarraysetops.py

```diff
@@ -303,14 +305,14 @@ def union1d(pda1: Union[pdarray, List[pdarray], Tuple[pdarray, ...]],
 
     Parameters
     ----------
-    pda1 : pdarray/List
-        Input array/List of input arrays
+    pda1 : pdarray/Sequence[pdarray, Strings, Categorical]
+        Input array/Sequence of groupable objectsx
```

Comment:
stray `x` in `objectsx`

Reply:
appears in other docstrings too

Reply:
updates all `objectsx` in docstring to `objects`.